File: src/BrunoVehicleHire.Domain/Entities/Vehicle.cs
namespace BrunoVehicleHire.Domain.Entities;

public class Vehicle
{
    public Guid Id { get; set; }
    public required string RegistrationNumber { get; set; } 
    public required string Make { get; set; }
    public required string Model { get; set; } 
    public required int Year { get; set; } 
    public bool IsDeleted { get; set; }
    public DateTime CreatedDate { get; set; }
}

File: bruno-vehicle-hire-web/src/models/Vehicle.ts
export interface Vehicle {
  id: string;
  registrationNumber: string;
  make: string;
  model: string;
  year: number;
  createdDate: string;
  deletedDate?: string;
}

export interface VehicleRequest {
  registrationNumber: string;
  make: string;
  model: string;
  year: number;
}
